.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,30 @@
+name: pythontestrunner
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Building ptr on python ${{ matrix.python-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.6, 3.7]
```

Comment:
Is 3.8 available?

Reply:
According to [this](https://github.com/actions/setup-python/issues/30) 3.8 isn’t supported yet.